lib/front_desk.rb

```diff
@@ -0,0 +1,105 @@
+require_relative 'reservation'
+require_relative 'date_range'
+
+module Hotel
+  class FrontDesk
+    attr_accessor :reservations, :rooms, :calendar
+
+    def initialize
+      @rooms = (1..20).map { |i| i }
```

Comment:
It looks like you're just using `map` to convert this to an array.  You can use `to_a` for that:

```suggestion
      @rooms = (1..20).to_a
```